src/sse/route-service.ts

```diff
@@ -2,13 +2,16 @@ import SSE from 'express-sse';
 import ChainwebEventService from './chainweb-event.js';
 import Logger from './logger.js';
 import ChainwebCutService from './chainweb-cut.js';
```

Comment:
Side note: I see you have a lot of `export default`s, it's considered best practice to only export named exports. This improves code readability and refactoring/navigation as the name stays the same across files.

Reply:
I'll leave this as a TODO as we want to expedite releasing